tests/subsys/event_manager/src/modules/test_oom.c

```diff
@@ -33,6 +33,24 @@ void sys_reboot(int type)
 	}
 }
 
+void *event_manager_alloc(size_t size)
+{
+	void *event = k_malloc(size);
+
+	if (unlikely(!event)) {
+		__ASSERT_NO_MSG(false);
```

Comment:
I guess we should not assert here, because we have a dedicated oom_error_handler. In the test we expect that this may happen.